<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>425</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>task1</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="colorButton">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>340</y>
      <width>111</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Сменить цвет</string>
    </property>
   </widget>
   <widget class="QPushButton" name="rotLeftButton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>370</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Влево</string>
    </property>
   </widget>
   <widget class="QPushButton" name="rotRightButton">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>370</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Вправо</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
